js/main.js

```diff
@@ -84,10 +121,30 @@ function switchView(viewName) {
 
 function handleViewNavigation(event) {
   var buttonDataView = event.target.getAttribute('data-view');
+  $formImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $photoUrl.setAttribute('value', '');
+  $photoTitle.setAttribute('value', '');
+  $comments.textContent = '';
   switchView(buttonDataView);
 }
 
-var $entriesButton = document.querySelector('.header-category');
+function edit(event) {
+
+  if (event.target.className === 'fas fa-pen') {
+    for (var i = 0; i < data.entries.length; i++) {
+      if (data.entries[i].entryId === parseInt(event.target.closest('li').getAttribute('data-entry-id'))) {
+        data.editing = data.entries[i];
+        $formImg.setAttribute('src', data.entries[i].photoUrlValue);
+        $photoUrl.setAttribute('value', data.entries[i].photoUrlValue);
+        $photoTitle.setAttribute('value', data.entries[i].photoTitleValue);
+        $comments.textContent = data.entries[i].commentsValue;
+      }
+    }
+    switchView(event.target.getAttribute('data-view'));
+    $form.reset();
```

Comment:
This isn't necessary here. In fact, I'm not sure how it's working to be honest, since the form control values you are setting on lines 138-140 should be getting cleared by this 🤔